homework3/semi-uber/Server/api/utilits/successResponse.js

```diff
@@ -0,0 +1,12 @@
+const log4js = require("log4js");
+const logger = log4js.getLogger();
+
+const success = (logMessage, data = null) => {
```

Comment:
you should add `status` as an optional argument because in some cases you may need the special status. And the same should be done for `errorHandler` utility function as well

Reply:
done